frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature

```diff
@@ -14,35 +14,36 @@ Feature: Create Application from Devfile
             Given user has created workload "nodejs-ex-git" with resource type "Deployment"
               And user is at the Topology page
              When user right clicks on topology empty graph
-              And user selects "From Devfile" option from Add to Project context menu
+              And user selects "Import from Git" option from Add to Project context menu
               And user enters Git Repo url "https://github.com/redhat-developer/devfile-sample" in Devfile page
-              And user enters Name as "node-bulletin-board-1" in DevFile page
-              And user clicks Create button on Devfile page
+              And user enters Name as "node-bulletin-board-1" in Import from Git page
```

Comment:
```suggestion
              And user enters workload name as "node-bulletin-board-1" in import from git form 
```

Reply:
Same for other such statements as well.

Reply:
I've kept `Import from Git ` as it is and changed the `page` to `form` because the form title is like this